src/main/java/no/unit/nva/cristin/projects/CristinApiClient.java

```diff
@@ -87,8 +87,7 @@ public ProjectsWrapper queryCristinProjectsIntoWrapperObjectWithAdditionalMetada
         long endRequestTime = System.currentTimeMillis();
 
         return new ProjectsWrapper()
-            .usingQueryParams(requestQueryParams)
-            .usingHeaders(response.headers())
+            .usingHeadersAndQueryParams(response.headers(), requestQueryParams)
```

Comment:
I think this is an indication of a problem in design. We are using a Builder like pattern, but we are mutating the object with every function. That is why we need to set the headers and query parameters at the same time.